iati/tests/about_functional_tests.py

```diff
@@ -0,0 +1,2 @@
+"""A module of functional tests for the about page and its sub pages."""
+import pytest
```

Comment:
Don't need to import, nothing is happening yet!

Would be picked up by linters as unused import (if we had them!)

Reply:
:+1: